Dockerfile.deploy.rhel

```diff
@@ -0,0 +1,19 @@
+FROM quay.io/openshiftio/rhel-base-golang:latest
```

Comment:
maybe we should start using multi-stage docker build https://docs.docker.com/develop/develop-images/multistage-build/

Reply:
@dipak-pawar it was not supported in OpenShift 3 because the version of the supported Docker daemon was too old. I'm not sure about OpenShift 4